critiquebrainz/frontend/views/search.py

```diff
@@ -24,6 +25,15 @@ def search_wrapper(query, type, offset=None):
             count, results = musicbrainz.search_labels(query, limit=RESULTS_LIMIT, offset=offset)
         elif type == "recording":
             count, results = musicbrainz.search_recordings(query, limit=RESULTS_LIMIT, offset=offset)
+        elif type == "bb_edition_group":
+            try:
+                count, results = musicbrainz.search_editon_group(query, limit=RESULTS_LIMIT, offset=offset)
+            except HTTPError:
+                raise BadRequest('Request failed while searching for edition groups.')
```

Comment:
This is good, but we should use an error code that indicates a server error (starting with 5) instead of BadRequest, which indicates a client error. Let's use `ServiceUnavailable`.

I think that we can return a single error message here in case of all of these errors. so let's just do

`exept (HTTPError, JSONDecodeError, Timeout):`

and raise the same error message